model-optimizer/mo/utils/ir_reader/extenders/strided_slice_extender.py

```diff
@@ -12,8 +14,12 @@ class StridedSlice_extender(Extender):
 
     @staticmethod
     def extend(op: Node):
+        input_shape = op.in_port(0).data.get_shape()
         for attr in StridedSlice.get_mask_names():
-            Extender.attr_to_list(op, attr)
+            if op[attr] != '':
+                Extender.attr_to_list(op, attr)
+            else:
+                op[attr] = np.zeros_like(input_shape)
```

Comment:
can you use just `int_array64([0])` so `input_shape` is not needed?

Reply:
Yes we can, we have an alignment of the masks in StridedSlice infer function